src/app/todoList/todoList.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component } from '@angular/core';
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Component({
+  selector: 'app-todoList',
+  templateUrl: './todoList.component.html',
+  styleUrls: ['./todoList.component.css']
+})
+
+export class TodoListComponent {
+  newTodoTitle: string = '';
+  todos: Array<Todo> = [];
+
+  remain() {
+    let il = 0;
```

Comment:
Zakładam, że to jest skrót od "ilość". W kodzie utrzymujemy nazewnictwo angielskie także niech ta zmienna niech ma sensowną nazwę.